guides/common/modules/proc_preparing-for-disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,44 @@
+[id="preparing-for-disaster-recovery-with-two-active-project-servers"]
+= Preparing for disaster recovery with two active {ProjectServer}s
+
+Create a second {ProjectServer} by restoring a backup of your first {ProjectServer}.
+Configure both servers to operate independently in their respective data centers, but ensure that their content, such as repositories or host configuration, does not drift apart over time.
+
+.Procedure
+. Back up your {ProjectServer}.
+For more information, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_admin[].
+. Restore the backup on a system that will serve as your other {ProjectServer}.
+For more information, see xref:restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[].
++
+[NOTE]
+====
+Each server must have a distinct hostname and IP address.
+This enables you to re-register hosts if one of the servers fails.
+====
+. Ensure that content on your servers remains consistent:
+** If you want both servers to act as sources for synchronization and content view creation, follow these guidelines to prevent content drift:
+*** Regularly synchronize repositories on both servers.
```

Comment:
Recommend use of Ansible collection?

Reply:
We have a link at the end of the procedure pointing to Foreman Ansible docs. I'm adding `redhat.satellite.repository_sync` and `redhat.satellite.sync_plan` as an extra tip here.